<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>curl Downloader</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/resources.qrc">
    <normaloff>:/Downloader/download.svg</normaloff>:/Downloader/download.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="m_scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>798</width>
         <height>556</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="m_scrollLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="m_spacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_file_to_download"/>
   <addaction name="actionApplication_settings"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
   <addaction name="actionMinimize"/>
   <addaction name="separator"/>
   <addaction name="actionExit_application"/>
  </widget>
  <action name="actionAdd_file_to_download">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/Downloader/add.svg</normaloff>:/Downloader/add.svg</iconset>
   </property>
   <property name="text">
    <string>Add file to download...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/Downloader/info.svg</normaloff>:/Downloader/info.svg</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
   <property name="toolTip">
    <string>About this application</string>
   </property>
  </action>
  <action name="actionExit_application">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/Downloader/exit.svg</normaloff>:/Downloader/exit.svg</iconset>
   </property>
   <property name="text">
    <string>Exit application</string>
   </property>
  </action>
  <action name="actionApplication_settings">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/Downloader/settings.svg</normaloff>:/Downloader/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Application settings...</string>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/Downloader/minimize.svg</normaloff>:/Downloader/minimize.svg</iconset>
   </property>
   <property name="text">
    <string>Minimize</string>
   </property>
   <property name="toolTip">
    <string>Minimizes the application to the tray menu.</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionMinimize</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
